lib/blocs/game_bloc.dart

```diff
@@ -68,6 +68,8 @@ class _Game {
 
   bool addLetter(String l) {
     if (word.value.length > Logic.MAX_WORD_LENGTH) {
+      _setTempMessage(Logic.wordTooLongMessage);
```

Comment:
I'm not an expert on BLoCs myself, and what belongs in business logic is subjective, but I think the message text can be considered business logic. But I do have a ToDo in the code to remove the UI act of displaying the message from GameBloc.